starter-code/scripts/article.js

```diff
@@ -29,6 +38,11 @@ Article.prototype.toHtml = function() {
     5. publication date. */
 
     // lots of $newArticle.find...  (look at jQuery $.find docs)
+    // $newArticle.attr('data-category', this.category);
+    $newArticle.find('h1').html(this.title);
+    $newArticle.find('a').attr('href', this.authorUrl);
```

Comment:
So if you look on live-server, this is currently not linking anything. So you must re-look at what you are targeting, maybe be more specific to the element?